scripts/install/opt/seagate/cortx/motr/conf/setup.yaml

```diff
@@ -29,7 +29,6 @@ motr:
     config:
         script: /usr/sbin/m0provision
         args:
-            - config
```

Comment:
[question]
If we have removed args : - config then does it make sense keeping m0provision with empty arguments ? can we remove this whole block ?

Reply:
This is kept for future, i.e after GA or for LDR R2

Reply:
Provisioner calls config section after deployment, but not after updates, thus i am making it a no-op here.